pkg/activator/net/revision_backends_test.go

```diff
@@ -55,15 +55,19 @@ const (
 	informerRestPeriod = 500 * time.Millisecond
 )
 
-func revision(revID types.NamespacedName, protocol networking.ProtocolType) *v1alpha1.Revision {
+// revisionCC1 - creates a revision with concurrency == 1.
+func revisionCC1(revID types.NamespacedName, protocol networking.ProtocolType) *v1alpha1.Revision {
```

Comment:
nit: Really worth it? Seems like you changed the calling site everywhere anyway.

Reply:
The changed one is in the `throttler_test.go`.